FrejaEidClient/src/test/java/com/verisec/frejaeid/client/http/CommonHttpTest.java

```diff
@@ -0,0 +1,92 @@
+package com.verisec.frejaeid.client.http;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import com.verisec.frejaeid.client.beans.common.RelyingPartyRequest;
+import com.verisec.frejaeid.client.beans.general.BasicUserInfo;
+import com.verisec.frejaeid.client.beans.general.RequestedAttributes;
+import com.verisec.frejaeid.client.beans.general.SsnUserInfo;
+import com.verisec.frejaeid.client.enums.Country;
+import com.verisec.frejaeid.client.util.JsonService;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
+import java.nio.charset.StandardCharsets;
+
+import org.apache.commons.codec.binary.Base64;
+import org.junit.After;
+import org.junit.Assert;
+
+public abstract class CommonHttpTest {
+
+    protected static final String REFERENCE = "123456789012345678";
+    protected static final String EMAIL = "eid.demo.verisec@gmail.com";
+    protected static final String SSN = "123455697887";
+    protected static final String RELYING_PARTY_ID = "relying_party_id";
+    protected static final SsnUserInfo SSN_USER_INFO = SsnUserInfo.create(Country.SWEDEN, SSN);
+    protected static final BasicUserInfo BASIC_USER_INFO = new BasicUserInfo("John", "Fante");
+    protected static final String CUSTOM_IDENTIFIER = "vejofan";
```

Comment:
Suggestion for the future: use values that reflect the purpose of the variable. Especially when it comes to similar-looking features like custom ID and organisation ID.

Also, since many of these variables are repeated in different tests. Can we keep them all in one place?

Reply:
That's a good idea. Let's make a task and resolve this in a separate pull request. The aim of this pull request is to add unit tests.